src/main/resources/logback.xml

```diff
@@ -4,7 +4,7 @@
             <pattern>%d{YYYY-MM-dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
```

Comment:
If all logs are in json format now, maybe this pattern should be made json-like?

Reply:
Good point, added json log appender